.github/workflows/ci_cron_daily.yml

```diff
@@ -0,0 +1,81 @@
+name: Daily Cron Tests
+
+on:
+  schedule:
+    # run at 6am UTC on Tue-Fri (complete tests are run every Monday)
+    - cron: '0 6 * * 2-5'
+  pull_request:
+    # We also want this workflow triggered if the 'Daily CI' label is added
+    # or present when PR is updated
+    types:
+      - synchronize
+      - labeled
+  push:
+    tags:
+      - '*'
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  TOXARGS: '-v'
+
+permissions:
+  contents: read
+
+jobs:
+  tests:
+    if: (github.repository == 'spacetelescope/slitlessutils' && (github.event_name == 'schedule' || github.event_name == 'push' || github.event_name == 'workflow_dispatch' || contains(github.event.pull_request.labels.*.name, 'Daily CI')))
+    name: ${{ matrix.prefix }} ${{ matrix.os }}, ${{ matrix.tox_env }}
+    runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.allow_failure }}
+    strategy:
+      matrix:
+        include:
+          - os: ubuntu-latest
+            python: '3.11'
+            tox_env: 'linkcheck'
+            allow_failure: false
+            prefix: ''
+
+          - os: ubuntu-latest
+            python: '3.11'
+            tox_env: 'py311-test-devdeps'
+            toxposargs: --remote-data=any
+            allow_failure: true
+            prefix: '(Allowed failure)'
+
+          - os: ubuntu-latest
+            python: '3.12-dev'
+            tox_env: 'py312-test-devdeps'
+            toxposargs: --remote-data=any
+            allow_failure: true
+            prefix: '(Allowed failure)'
+
+    steps:
+    - name: Check out repository
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+    - name: Set up Python ${{ matrix.python }}
+      uses: actions/setup-python@v4
+      with:
+        python-version: ${{ matrix.python }}
+    - name: Install base dependencies
+      run: python -m pip install --upgrade pip setuptools tox
+    - name: Print Python, pip, setuptools, and tox versions
```

Comment:
Why not use `pip freeze`?

Reply:
`pip freeze` doesn't include those packages, e.g.:
https://github.com/spacetelescope/slitlessutils/actions/runs/6317086199/job/17153095889#step:6:24